modules/dxp/apps/multi-factor-authentication/multi-factor-authentication-timebased-otp-web/src/main/java/com/liferay/multi/factor/authentication/timebased/otp/web/internal/display/context/MFATimeBasedOTPCheckerDisplayContext.java

```diff
@@ -63,11 +67,13 @@ public Map<String, Object> getContext() throws PortalException {
 					MFATimeBasedOTPWebKeys.MFA_TIME_BASED_OTP_DIGITS))
 		).put(
 			"issuer",
-			HtmlUtil.escapeJS(
-				GetterUtil.getString(
-					_httpServletRequest.getAttribute(
-						MFATimeBasedOTPWebKeys.
-							MFA_TIME_BASED_OTP_COMPANY_NAME)))
+			StringUtil.replace(
+				HtmlUtil.escapeJS(
+					GetterUtil.getString(
+						_httpServletRequest.getAttribute(
+							MFATimeBasedOTPWebKeys.
+								MFA_TIME_BASED_OTP_COMPANY_NAME))),
+				"\\x20", " ")
```

Comment:
Is `\\x20` the only character code we need to translate back to ascii?  Are there any other characters?  Is there a util of some sort we can use instead?